configuration/concepts/operative_progress.csv

```diff
@@ -0,0 +1,21 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members,Units
+a957ff9c-80f1-4f31-8937-3bef0cfb8c0e,,Pre-Op Diagnosis,,Diagnóstico preoperatorio,Diagnóstico preoperatorio,,Misc,Text,,,
+f348e4b5-4124-4169-966a-2dff2a944566,,post-Op Diagnosis,,diagnóstico postoperatorio,diagnóstico postoperatorio,,Misc,Text,,,
```

Comment:
Lets use the FSN as 'Post-Operative Diagnosis'

Reply:
Done